blog/_posts/2022-11-04-scala-3.2.1-released.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.2.1 released!
+---
+
+[Scala 3.2.1](https://github.com/lampepfl/dotty/releases/tag/3.2.1) is now available!
+
+This version is a bug-fixing release focusing mainly but not exclusively on solving rare issues with code not compiling after migration to 3.2.0. This release also brings two notable enhancements:
+
+- You can use published versions of the compiler to experiment with [capture checking](https://docs.scala-lang.org/scala3/reference/experimental/cc.html). You are no longer required to build the compiler from the sources from the experimental branch. Note that the capture-checking feature is still in an early experimental stage and needs lots of work and time before it becomes stable.
```

Comment:
We should explicitly write how to enable capture checking. Maybe even give some examples?